features/step_definitions/common_steps.rb

```diff
@@ -1,6 +1,18 @@
 require 'uri'
 require 'securerandom'
 
+def find_checkboxes_by_name(name, options={})
+  return all_fields(name, options.merge({type: 'checkbox'}))
+end
+
+def find_radios_by_name(name, options={})
+  return all_fields(name, options.merge({type: 'radio'}))
+end
+
+def find_radio_by_name(name, options={})
```

Comment:
Are we likely to ever use this `find_radio_by_name` method at all?  It's not used here, and the approach of finding all radios with a name and checking a random one seems good to me.